src/configureWorkspace/configurePython.ts

```diff
@@ -60,6 +60,20 @@ services:
       dockerfile: Dockerfile
 $ports$`;
 
+const dockerComposeDebugfile = `version: '3.4'
+
+services:
+  $service_name$:
+    image: $service_name$
+    build:
+      context: .
+      dockerfile: Dockerfile
+    entrypoint: /bin/bash
+    command: -c "pip install debugpy -t /tmp && python /tmp/debugpy --wait-for-client --listen 0.0.0.0:5678 $cmd$"
```

Comment:
Is `--wait-for-client` desirable in this scenario? In contrast, starting a .NET Core app via `docker-compose.debug.yml` will allow the application itself to start and go.

Reply:
Hmm, that's a valid point. I don't really have a strong opinion, I would have thought that the user might want to debug startup logic as well. Did we ever have specific feedback on this?

Reply:
Nothing specific, but our answer for debugging startup logic is "F5", not debug-compose.yml.

Reply:
The problem with F5 is "what if I need more containers to start as well"? Almost every Python example that I saw has at least 1 more container that's usually a database.

Reply:
Same problem exists for .NET :wink: